types/2020-03-02/Issuing/Cardholders.d.ts

```diff
@@ -6116,7 +6116,7 @@ declare module 'stripe' {
         status?: CardholderListParams.Status;
 
         /**
-         * Only return cardholders that have the given type. One of `individual`, `business_entity`, or `company`.
+         * Only return cardholders that have the given type. One of `individual` or `company`.
```

Comment:
Just calling out that this docstring and the one above now disagree with the types. I think that's probably fine (just implies that `business_entity` is rare/unexpected, but possible).